app/assets/javascripts/editor/editor.js.erb

```diff
@@ -828,6 +840,28 @@ var CodeOceanEditor = {
         });
     },
 
+    showScoringFailureMessage: function () {
+        $.flash.warning({
+            icon: ['fa-solid', 'fa-triangle-exclamation'],
+            text: I18n.t('exercises.submit.failure')
+        });
+    },
+
+    showScoringTooLateMessage: function () {
+        $.flash.warning({
+            icon: ['fa-solid', 'fa-triangle-exclamation'],
+            text: I18n.t('exercises.submit.too_late')
```

Comment:
We might want to update this text as well, making it a bit more verbose. It's not just about receiving the submission, but also about scoring it and transmitting this score to the e-learning platform. However, we haven't transmitted the achieved score, but rather X points.